.github/workflows/npm-test.yml

```diff
@@ -1,35 +0,0 @@
----
-name: Pull Request NPM Checks
-on: [pull_request]
-concurrency:
-  group: ci-${{ github.ref }}
-  cancel-in-progress: true
-jobs:
-  npm-test:
-    name: NPM test
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Sources
-        uses: actions/checkout@v2
-      - name: Prepare Node.js
-        uses: actions/setup-node@v1
-        with:
-          node-version: '12.x'
-      - name: Run `npm install`
-        run: npm install
-      - name: Run `npm test`
-        run: npm test
-  npm-lint:
-    name: NPM lint
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout Sources
-        uses: actions/checkout@v2
-      - name: Prepare Node.js
-        uses: actions/setup-node@v1
-        with:
-          node-version: '12.x'
-      - name: Run `npm install`
-        run: npm install
-      - name: Run `npm run lint`
-        run: npm run lint
```

Comment:
This file was deleted, but lint check wasn't moved to the other workflow

Reply:
@ivangergo3 as @dimko maybe you should move the lint check to the new file!
If you check, the checks fail on lint! 😄

Reply:
As @dimko mentioned, maybe the check for lint should be moved to the new file as well! 😄